create_daily_notion_page.py

```
@@ -44,24 +52,151 @@ def build_children(tasks):
+        return True
+    
+    print(f"📋 Found {len(child_pages)} child pages, new page is at position {new_page_index + 1}")
+    print("🔄 Reordering pages to move new page to top...")
+    
+    # Archive all pages except the new one
+    pages_to_reorder = []
+    for i, page in enumerate(child_pages):
+        if page["id"] != page_id:
+            # Archive this page
+            archive_url = f"{PAGES_URL}/{page['id']}"
+            archive_response = requests.patch(
```

Ruff (0.14.10)

Probable use of `requests` call without timeout

(S113)